mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -573,7 +573,17 @@ CLASS net/minecraft/class_1297 net/minecraft/entity/Entity
 	METHOD method_5721 isTouchingWaterOrRain ()Z
 	METHOD method_5722 isTeammate (Lnet/minecraft/class_1297;)Z
 		ARG 1 other
-	METHOD method_5723 dealDamage (Lnet/minecraft/class_1309;Lnet/minecraft/class_1297;)V
+	METHOD method_5723 applyDamageEffects (Lnet/minecraft/class_1309;Lnet/minecraft/class_1297;)V
+		COMMENT Applies damage effects to {@code attacker} or {@code target}.
+		COMMENT
+		COMMENT <p>Called when {@code attacker} damages {@code target}. The attacker is
+		COMMENT usually this entity, but may be a projectile's owner entity.
+		COMMENT
+		COMMENT <p>Used to apply enchantment effects such as thorn or slowness of bane
+		COMMENT of arthropods.
+		COMMENT
+		COMMENT @implNote Although this method is non-static, {@code this} keyword is
+		COMMENT not used anywhere in this method.
```

Comment:
If this should be documented, I think it should be processed similar to #2395.

Reply:
Also not using `this` doesn't mean having this method as non-static is pointless: the subclasses can override this instance method while static methods can't be overridden.